app/src/main/java/com/teamwss/websoso/data/di/ApiModule.kt

```diff
@@ -0,0 +1,37 @@
+package com.teamwss.websoso.data.di
+
+import com.teamwss.websoso.data.remote.api.FeedApi
+import com.teamwss.websoso.data.remote.api.NovelApi
+import com.teamwss.websoso.data.remote.api.UserNovelApi
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object ApiModule {
+
+    val feedApi = NetworkModule.provideApi<FeedApi>(
+        NetworkModule.provideRetrofit(
+            NetworkModule.provideLogOkHttpClient()
+        )
+    )
+
+    @Provides
+    @Singleton
+    fun provideNovelApi() = NetworkModule.provideApi<NovelApi>(
+        NetworkModule.provideRetrofit(
+            NetworkModule.provideLogOkHttpClient()
+        )
+    )
+
```

Comment:
r: 
```suggestion

    @Provides
    @Singleton
    fun providesNovelApi(retrofit: Retrofit): NovelApi = retrofit.create()
```
이렇게만 써도 된단 사실.. ㅎㅎ  
Retrofit 라이브러리에서 create타입만 명시해주면 알아서 서비스 객체를 만들어주는 확장함수를 제공합니다.

Reply:
굉장히 편리한 친구가 있었군요 개쩔탱 👍

Reply:
저도 이거 리뷰 받고 알았습니다